rules/autoconsent/b-cookie.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "b-cookie",
+    "vendorUrl": "https://unknown",
```

Comment:
It’s an optional field so shouldn’t we omit `vendorUrl` for unknown CMPs?

Reply:
this line just came from the create-rule template. It's not used by the code, so I think we should just strip it out during build time